backend/src/main/java/ai/verta/modeldb/versioning/blob/visitors/Visitor.java

```diff
@@ -3,101 +3,103 @@
 import ai.verta.modeldb.ModelDBException;
 import ai.verta.modeldb.versioning.autogenerated._public.modeldb.versioning.model.*;
 import ai.verta.modeldb.versioning.blob.diff.Utils;
+
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class Visitor {
-  public void preVisitListOfAutogenBlob(List<AutogenBlob> lst) throws ModelDBException {
+  public void preVisitListOfAutogenBlobDiff(List<AutogenBlobDiff> lst) throws ModelDBException {
     if (lst == null) return;
-    for (AutogenBlob val : lst) {
-      preVisitAutogenBlob(val);
+    for (AutogenBlobDiff val : lst) {
```

Comment:
I do not understand why this file changed

Reply:
I believe this file is auto-generated (it might be missing the header).

Reply:
for some reason I have different result than in master when I run ./swagger_codegen.sh . Did I miss something here?

Reply:
It mostly imports and method order.

I did just a small change in generation template. But when I don't do any changes it's still different

Reply:
@conradoverta : could you review the autogen files and the process. the code merged from #425 makes sense to me otherwise.